StayGo.Domain/Models/Customer.cs

```diff
@@ -0,0 +1,9 @@
+namespace StayGo.Domain.Models
+{
+    public class Customer
+    {
+        private int Id { get; set; }
+        private string Name { get; set; }
+        private string DateBirth { get; set; }        
```

Comment:
Why DateBirth is string ? 
Have to be DateTime

Reply:
Something I didn't immediately think about it